doc/users/whats_new.rst

```diff
@@ -31,27 +31,27 @@ The polar axes transforms have been greatly re-factored to allow for more
 customization of view limits and tick labelling. Additional options for view
 limits allow for creating an annulus, a sector, or some combination of the two.
 
-The :meth:`~matplotlib.axes.projections.polar.PolarAxes.set_rorigin` method may
+The :meth:`~matplotlib.projections.polar.PolarAxes.set_rorigin` method may
```

Comment:
I think you can get away (here and below) with omitting the :meth: because the current default role is :py:obj:.

Reply:
Quite likely; I was a bit lazy and didn't change the existing ones, but generally didn't use any roles when adding the new links. There might be a slight difference with adding `()` for `:func:` and `:meth:`, though?

Reply:
http://www.sphinx-doc.org/en/stable/domains.html#role-py:func
http://www.sphinx-doc.org/en/stable/config.html#confval-add_function_parentheses

Not sure whether it is set in conf.py right now, or whether py:obj respecsts that setting.  If it doesn't we can just set add_function_parentheses to False.

Of course it's fine if you decide you don't want to mess with the markup, this is already an improvement...

Reply:
edit: https://github.com/sphinx-doc/sphinx/issues/171 suggests () are added correctly.

Reply:
When I tried the two options you gave below, the one with the `:meth:` role has parentheses while the one without doesn't.

Reply:
Probably worth reporting as a Sphinx bug then?